service/matchers_test.go

```diff
@@ -0,0 +1,51 @@
+package service
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsRunning(t *testing.T) {
+	assert := assert.New(t)
+
+	svcs := []struct {
+		name        string
+		sysInitType string
+	}{
+		{"docker", "upstart"},
+		{"docker", "systemd"},
+	}
+
+	statuses := []string{"running", "stopped"}
+	for _, status := range statuses {
+		for _, svc := range svcs {
+			currentDir, err := os.Getwd()
+			assert.NoError(err)
+			fixturesPath := path.Join(currentDir, "test-fixtures",
```

Comment:
```
file := fmt.Sprintf("%s-%s.out", svc.sysInitType, status)
fixturesPath := path.Join(currentDir, "test-fixtures", file)
```

Or:

```
fixturesPath := getFixturePath(svc.sysInitType, status)
```

With the latest you will get rid of `currentDir :=...` and the other `assert`.